packages/spark-core/tools/_mixins.scss

```diff
@@ -52,10 +52,10 @@
   }
 
   &::before {
-    margin-bottom: calc(-#{$dynamic-top-crop}em + #{$top-adjustment});
+    margin-bottom: calc(-#{$dynamic-top-crop}em + #{$top-adjustment}px);
```

Comment:
Do we want this to be a hot-fix on master branch?

Reply:
yeah for sure, good catch